<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="avaLabel">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>59</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ava</string>
    </property>
   </widget>
   <widget class="QLabel" name="avaImg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>330</width>
      <height>330</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/img/Ava.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/avaImg/과제/img/Ava.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>749</x>
      <y>530</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Start / Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replayButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Replay</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="recordingDisplay">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="numDisplay">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ava_img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>578</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>842</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>717</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replayButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>replay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>exit()</slot>
  <slot>next()</slot>
  <slot>replay()</slot>
 </slots>
</ui>
